src/main/java/dev/jbang/cli/Export.java

```diff
@@ -653,20 +654,23 @@ void renderBuildFile(BuildContext ctx, Path projectDir, String fullClassName) th
 		Template template = engine.getTemplate(templateRef);
 		if (template == null)
 			throw new ExitException(EXIT_INVALID_INPUT, "Could not locate template named: '" + templateRef + "'");
+		boolean isKotlin = ctx.getProject().getMainClass().endsWith("Kt");
```

Comment:
should be updated to use same test as above.

Reply:
Fixed